common.sh

```diff
@@ -136,6 +136,10 @@ fi
 
 export OPENSHIFT_RELEASE_TAG=$(echo $OPENSHIFT_RELEASE_IMAGE | sed -E 's/[[:alnum:]\/.-]*release.*://')
 
+if [[ "$OPENSHIFT_VERSION" != "4.3" ]]; then
```

Comment:
This var isn't set in OCP CI, which might not be a problem but I'd like to make sure this works. Or we could set OPENSHIFT_VERSION explicitly to "" in CI. We don't test 4.3 there anyway

Reply:
thanks, done.